common/djangoapps/student/models.py

```diff
@@ -1442,6 +1447,16 @@ def enroll(cls, user, course_key, mode=None, check_access=False):
 
         Also emits relevant events for analytics purposes.
         """
+        if hasattr(course_key, 'ccx') and configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False):
```

Comment:
@elvinquero, to check if it is a CCX course, I think you can use the function `is_ccx_course` like this:
```suggestion
        if is_ccx_course(course_key) and configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False):
```